less/index.less

```diff
@@ -54,6 +54,448 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: @cent;
 }
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: @cent;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+html,
+body {
+  height: @cent;
+}
+
+// VARIABLES
+@bg-color:  teal;
+@size: 20px;
+@size10: 10px;
+@size30: 30px;
+@direction: flex;
+@location: center;
+@size60: 60px;
+@size100: 100px;
+@cent: 100%;
+@small: bold;
+@more: bolder;
+@cent8: 80%;
+@size1: 1px;
+@gap: space-between;
+@mobile: 500px;
+@size80: 80px;
+@size40: 40px;
+@size15: 15px;
+@orientation: row;
+@colour: blue;
+
+// MIXINS
+.anchor{
+	font-size: @size;
+	color: @colour;
+}
+
+.free{
+	display: @direction;
+	flex-direction: @orientation ;
+}
+
+
```

Comment:
Please use a document formatter to tighten up your code and eliminate any extra lines and spacing.